redash/models/__init__.py

```diff
@@ -198,21 +198,25 @@ def delete(self):
         return res
 
     def get_schema(self, refresh=False):
-        key = "data_source:schema:{}".format(self.id)
```

Comment:
Is removing this redis caching of schema information intentional? Is there a performance impact?

Reply:
Thanks for pointing this out @washort!

It was intentional because from what I recall back in the Berlin work-week, I think @arikfr  was saying he felt that using redis to store schema was a bit of a hack and he would prefer it stored in a table. Of course, we could be storing the data in tables and have additional caching for performance, but I felt this added complexity of maintaining both a cache and tables for the same data was perhaps not worth the performance gain.

I did a quick test on my machine and with 5 runs of the old vs. the new `get_schema()` function, the redis one averages 7.2ms per call and this one (from this pr) averages 44ms per call. It's a big relative difference, but 44ms isn't so bad. Though of course this could be worse in different scenarios, e.g. slower network/machine or more data. I suppose I will defer this decision to @arikfr

Reply:
Great, just curious.